lib/index.js

```diff
@@ -83,9 +84,20 @@ Mixpanel.prototype.identify = function(identify, fn){
     verbose: 1        // make sure that we get a valid response
   };
 
+  var traits = formatTraits(identify);
+
+  if (!this.settings.setAllTraitsByDefault) {
```

Comment:
@tsholmes what's the impact of this addition? is this an augmentative change in the data they would see or breaking?